### DISCUSSION TITLE
Personal Notes - Where and how are they stored?

### DISCUSSION THREAD

**Discussion by tzinm** [javikr/sofatime#63] [Q&A] [answered]
Trakt has the possibility to add private notes, but only to [VIP users](https://trakt.tv/vip). When you create a new personal note and synchronize Sofa Time with Trakt you can see there is no private note created in the Trakt profile.

What seems strange for me is that when I sign out of the Trakt profile in the app, the personal notes disappear. When I sign in again, the notes appear again.

**How are personal notes managed in the app?**

I imagine that if a user decides to use an application other than Sofa Tim, the user will lose all personal notes. **Is that correct?**

**Comment by javikr** [comment, marked as answer]
Hi there!

Yes, you're right. Personal Notes in Sofa Time were implemented before Trakt and currently are managed only in Sofa Time, but we have plans to migrate them to Trakt ASAP.
